deploy/kubeturbo-operator/test.sh

```diff
@@ -2,48 +2,56 @@
 
 SCRIPT_DIR="$(cd "$(dirname $0)" && pwd)"
 ERR_LOG=$(mktemp --suffix _kube.errlog)
-WAIT_FOR_DEPLOYMENT=20
+WAIT_FOR_DEPLOYMENT=30
 OPERATOR_IMAGE_VERSION="8.9.5-SNAPSHOT"
 OPERATOR_IMAGE="icr.io\/cpopen\/kubeturbo-operator:${OPERATOR_IMAGE_VERSION}"
+KUBETURBO_IMAGE_VERSION="8.9.5-SNAPSHOT"
```

Comment:
I don't think it's right to hardcode this version. Hardcoding the version means we always test this `8.9.5` version. It loses the point of the testing.  This version should be configurable or passed from the command line.

Reply:
Updated PR to use the existing VERSION env var instead of hardcoding it in test.sh. You can now overwrite the specific version like this: `make test VERSION=8.9.5-SNAPSHOT`

Reply:
In my other PR, I read the version and the image registry from the environment so that the Jenkins build can pass the version to the script. While the hardcoded version can be treated as the safeguard to make sure the script passes with a certain version.